request/en.txt

```diff
@@ -0,0 +1,65 @@
+Dear ○○,
+
+This is a staff of a Ruby 25th anniversary event.
+Ruby was born in 24th Feb, 1993, so will be 25th birthday.
+We held a event that celebrating and talking about Ruby's past, now and future, on 24th Feb. 2018, in Tokyo.
```

Comment:
We will hold an event celebrating Ruby, and talking about Ruby's past, present, and future on the 24th of February 2018 in Tokyo.